js/user.js

```diff
@@ -1,25 +1,76 @@
-function main() {
-  var db = new Kinto({remote: "https://kinto.dev.mozaws.net/v1"});
-  var tasks = db.collection("users");
-
-  document.getElementById("form")
-    .addEventListener("submit", function(event) {
-      event.preventDefault();
-      tasks.create({
-        title: event.target.title.value,
-        url: event.target.url.value,
-        done: false
-      })
-      .then(function(res) {
-        event.target.title.value = "";
-        event.target.url.value = "";
-        event.target.title.focus();
-      })
-      .catch(function(err) {
-        console.error(err);
-      });
+var central_repository_server = "https://kinto.dev.mozaws.net/v1";
+function authenticate(){
+
+    var storageServer = document.getElementById("kinto_server").value;
+    if (storageServer == ''){
+      storageServer = "https://kinto.dev.mozaws.net/v1"  //default
```

Comment:
kinto.dev.mozaws.net should become a global constant so that we can change the default value easily.